unittests/inference.py

```diff
@@ -37,15 +37,15 @@ def infer_file(cls, path):
         t = ast.parse(r.read())
         r.close()
 
-        analyzer = PreAnalyzer(t)
+        analyzer = PreAnalyzer(t, "tests/inference")
```

Comment:
Wait, do we still not have a test suite that runs all the files we have in a row??